### ISSUE TITLE
Consistent Error: Element not found

### ISSUE THREAD

**Issue by vexersa** [Ashish-Github193/facebook-ads-library#1]
I keep getting the following error: 

DevTools listening on ws://127.0.0.1:58688/devtools/browser/39b6e632-5501-499e-8ec0-daa9667b56a1
[90936:79460:0628/082904.797:ERROR:device_event_log_impl.cc(222)] [08:29:04.796] USB: usb_service_win.cc:104 SetupDiGetDeviceProperty({{A45C254E-DF1C-4EFD-8020-67D146A850E0}, 6}) failed: Element not found. (0x490)
ERROR:root:Error occurred while checking the main division.

**Comment by Ashish-Github193** [comment]
Could you please comment on the '--headless=new' line in the following file and verify if the elements are loading? Sometimes, the ads library may block the IP if it deems it suspicious.
Path: utils/utils_initialise_function.py

If that is not the issue, then please let me know.

**Comment by vexersa** [comment]
Getting the following: 
[WDM] - Downloading: 100%|██████████████████████████████████████████████████████████████████████████████| 6.30M/6.30M [00:00<00:00, 77.7MB/s]

DevTools listening on ws://127.0.0.1:51045/devtools/browser/549adecc-42b9-41fb-b040-73d4f644d803

DevTools listening on ws://127.0.0.1:51088/devtools/browser/16128113-8c7e-4b55-b64c-11514057b691
[63376:80272:0629/095452.493:ERROR:device_event_log_impl.cc(222)] [09:54:52.493] USB: usb_service_win.cc:104 SetupDiGetDeviceProperty({{A45C254E-DF1C-4EFD-8020-67D146A850E0}, 6}) failed: Element not found. (0x490)

DevTools listening on ws://127.0.0.1:51145/devtools/browser/fddadf65-a5a3-4137-a800-a92b2db656bc
[109020:115556:0629/095515.294:ERROR:device_event_log_impl.cc(222)] [09:55:15.295] USB: usb_service_win.cc:104 SetupDiGetDeviceProperty({{A45C254E-DF1C-4EFD-8020-67D146A850E0}, 6}) failed: Element not found. (0x490)

DevTools listening on ws://127.0.0.1:51206/devtools/browser/a2d88c75-938c-4574-ab63-bf32c931262f
[45820:115268:0629/095537.947:ERROR:device_event_log_impl.cc(222)] [09:55:37.948] USB: usb_service_win.cc:104 SetupDiGetDeviceProperty({{A45C254E-DF1C-4EFD-8020-67D146A850E0}, 6}) failed: Element not found. (0x490)

DevTools listening on ws://127.0.0.1:51241/devtools/browser/60bf2e27-bb7f-4ec4-b4ce-6d237945e025
[111952:108920:0629/095546.061:ERROR:device_event_log_impl.cc(222)] [09:55:46.061] USB: usb_service_win.cc:104 SetupDiGetDeviceProperty({{A45C254E-DF1C-4EFD-8020-67D146A850E0}, 6}) failed: Element not found. (0x490)
ERROR:root:Error occurred while checking the main division.

It has come to our attention that your current activity on Facebook is proceeding
at an accelerated rate. As a result, your IP address has been blocked by Facebook
due to the excessive number of page requests made within a very short period. Please note
that this block is temporary in nature, which means that your IP will remain restricted
for a duration ranging from 8 to 24 hours.

To confirm the status of your IP address, we recommend running the scraper tool again on the
following day. If, despite the passage of time, you still encounter the same issue and find that
your IP address remains blocked, please get in touch with developer team.
Done.




The browser window opens and it appears to be working, but there are no saved ads and I get the above in the console.

**Comment by Ashish-Github193** [comment]
Check it out... I've made some changes; therefore it may work now.

**Comment by vexersa** [comment]
Thanks! Works now. 
Two questions (let me know if I should open a new issue for these 🙏)
1. Are images/videos downloaded? 
2. Is it expected that the application deletes previously run searches? I would expect it to just keep previous search's .csv's by default.

**Comment by Ashish-Github193** [comment]
If you're lazy enough, you can create a new issue... but personally, I don't have any problem with this one either. In short, it completely depends on you. If you believe creating a new issue would be beneficial for you, feel free to do so. I don't have any issues with that.

1. No, not yet. However, you can create a new scraper plugin if you would like. I'm not very skilled in code organization, but I've designed it in a way that you can add your own scraping plugin, so you can create one for yourself. Alternatively, let me know, and I'll create it for you. I'm busy on weekdays, so I'll work on it during the weekends.
2. It's a bit unclear what you're trying to say here:
    Maybe you're suggesting that after completing one scrape, it should not delete the scraped data. If that's the case, I haven't added that feature yet, so no, it's not possible to keep the old file. It will override the scraped data. However, you can move your CSV file from the data folder to wherever you want to save it before you start another scraping task. Next time, the scraper will still be able to create a new one automatically.

Note: Please make sure not to move the all_country_code.csv file. Other than that, it's all yours.

**Comment by vexersa** [comment]
Cool!

I think that I can take a stab at writing the updpates myself and push some
commits?

**Comment by Ashish-Github193** [comment]
I can create them for you as well, but please note that it will take time as I often get busy. Therefore, it won't be possible for me to do it quickly. So, if you're short on time, you can go ahead and create and commit the changes yourself.

**Comment by vexersa** [comment]
Cool :)
